The render delegate fails to build using gcc 4.8.5

When building the render delegate using gcc 4.8.5, it fails with the following error:

```
render_delegate/camera.cpp: In lambda function:
render_delegate/camera.cpp:102:14: error: converting to ‘std::tuple<foundryInternal_v19_11_fn__pxrReserved__::TfToken, AtString>’ from initializer list would use explicit constructor ‘constexpr std::tuple<_T1, _T2>::tuple(_U1&&, _U2&&) [with _U1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken&; _U2 = const AtString&; <template-parameter-2-3> = void; _T1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken; _T2 = AtString]’
             }};
              ^
render_delegate/camera.cpp:102:14: error: converting to ‘std::tuple<foundryInternal_v19_11_fn__pxrReserved__::TfToken, AtString>’ from initializer list would use explicit constructor ‘constexpr std::tuple<_T1, _T2>::tuple(_U1&&, _U2&&) [with _U1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken&; _U2 = const AtString&; <template-parameter-2-3> = void; _T1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken; _T2 = AtString]’
render_delegate/camera.cpp:102:14: error: converting to ‘std::tuple<foundryInternal_v19_11_fn__pxrReserved__::TfToken, AtString>’ from initializer list would use explicit constructor ‘constexpr std::tuple<_T1, _T2>::tuple(_U1&&, _U2&&) [with _U1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken&; _U2 = const AtString&; <template-parameter-2-3> = void; _T1 = foundryInternal_v19_11_fn__pxrReserved__::TfToken; _T2 = AtString]’
```

--- a/render_delegate/camera.cpp
+++ b/render_delegate/camera.cpp
@@ -95,11 +95,10 @@ void HdArnoldCamera::Sync(HdSceneDelegate* sceneDelegate, HdRenderParam* renderP
         const static CameraParamMap cameraParams = []() -> CameraParamMap {
             // Exposure seems to be part of the UsdGeom schema but not exposed on the Solaris camera lop. We look for
             // both the primvar and the built-in attribute, and preferring the primvar over the built-in attribute.
-            CameraParamMap ret{
-                {_tokens->exposure, str::exposure},
-                {HdCameraTokens->shutterOpen, str::shutter_start},
-                {HdCameraTokens->shutterClose, str::shutter_end},
-            };
+            CameraParamMap ret;
+            ret.emplace_back(_tokens->exposure, str::exposure);
+            ret.emplace_back(HdCameraTokens->shutterOpen, str::shutter_start);
+            ret.emplace_back(HdCameraTokens->shutterClose, str::shutter_end);
             for (const auto* paramName :
                  {"exposure", "radial_distortion", "radial_distortion_type", "shutter_type", "rolling_shutter",
                   "rolling_shutter_duration", "aperture_blades", "aperture_rotation", "aperture_blade_curvature",

